gate-web/src/main/groovy/com/netflix/spinnaker/gate/security/userdetailsservice/google/GoogleDirectoryUserProviderConfig.groovy

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2016 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.gate.security.userdetailsservice.google
+
+import org.springframework.boot.context.properties.ConfigurationProperties
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+@ConfigurationProperties("auth.groupMembership.google")
+class GoogleDirectoryUserProviderConfig {
+
+  String serviceAccountEmail
+
+  String credentialPath
+
+  String adminUsername
```

Comment:
Add some documentation to this field describing the requirement for an admin account.